src/hooks/useVaultLocationAudit.ts

```diff
@@ -109,13 +111,18 @@ export default function useVaultLocationAudit(): UseVaultLocationAudit {
               ...itemAudit,
               activityDetail: `Removed from ${
                 vaultLocations?.[selectedItems[itemId]?.vaultLocation].name
-              } ${editRemarks !== undefined ? `Remarks: ${editRemarks}` : ''}`
+              } ${editRemarks !== undefined ? `Remarks: ${editRemarks}` : ''}`,
+              subjectId:
```

Comment:
Hello @TahaKhanAbdalli - is there any chance we can just use `selectedItems[itemId]?.vaultLocation` in the next line?